src/bio/bio_xstream.c

```diff
@@ -433,16 +183,17 @@ bio_nvme_init(const char *nvme_conf, int shm_id, int mem_size,
 	nvme_glb.bd_shm_id = shm_id;
 	nvme_glb.bd_mem_size = mem_size;
 
+	nvme_glb.bd_nvme_conf = nvme_conf;
+
 	rc = bio_spdk_env_init();
 	if (rc)
 		goto free_conf;
 
 	return 0;
 
+
 free_conf:
```

Comment:
Minor: this 'free_conf' label could be removed.

Reply:
yep